packages/jest-types/src/Config.ts

```diff
@@ -12,7 +12,104 @@ import type {SnapshotFormat} from '@jest/schemas';
 
 type CoverageProvider = 'babel' | 'v8';
 
-type Timers = 'real' | 'fake' | 'modern' | 'legacy';
+export type FakeableTimerAPIs =
+  | 'setImmediate'
+  | 'clearImmediate'
+  | 'setInterval'
+  | 'clearInterval'
+  | 'setTimeout'
+  | 'clearTimeout'
+  | 'requestAnimationFrame'
+  | 'cancelAnimationFrame'
+  | 'requestIdleCallback'
+  | 'cancelIdleCallback'
+  | 'Date'
+  | 'hrtime'
+  | 'nextTick'
+  | 'performance';
+// | 'queueMicrotask'; // ?
```

Comment:
Would it make sense to include `queueMicrotask`? It is absent in `@types/sinonjs__fake-timers`.